README.md

```diff
@@ -2,35 +2,84 @@
 
 > Facebook OAuth2 strategy for Überauth.
 
-### Setup
+## Installation
 
-Include the provider in your configuration for Überauth:
+1. Setup your application at [Facebook Developers](https://developers.facebook.com).
 
-```elixir
-config :ueberauth, Ueberauth,
-  providers: [
-    facebook: [ { Ueberauth.Strategy.Facebook, [] } ]
-  ]
-```
+1. Add `:ueberauth_facebook` to your list of dependencies in `mix.exs`:
 
-Then configure your provider:
+    ```elixir
+    def deps do
+      [{:ueberauth_facebook, "~> 0.1"}]
+    end
+    ```
 
-```elixir
-config :ueberauth, Ueberauth.Strategy.Facebook.OAuth,
-  client_id: System.get_env("FACEBOOK_APP_ID"),
-  client_secret: System.get_env("FACEBOOK_APP_SECRET")
-```
+1. Add the strategy to your applications:
 
-For an example implementation see the [Überauth Example](https://github.com/doomspork/ueberauth_example) application.
+    ```elixir
+    def application do
+      [applications: [:ueberauth_facebook]]
+    end
+    ```
 
-## Installation
+1. Add Facebook to your Überauth configuration:
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed as:
+    ```elixir
+    config :ueberauth, Ueberauth,
+      providers: [
+        facebook: [{Ueberauth.Strategy.Facebook, []}]
+      ]
+    ```
 
-1. Add `:ueberauth_facebook` to your list of dependencies in `mix.exs`:
+1.  Update your provider configuration:
 
     ```elixir
-    def deps do
-      [{:ueberauth_facebook, "~> 0.1"}]
+    config :ueberauth, Ueberauth.Strategy.Facebook.OAuth,
+      client_id: System.get_env("FACEBOOK_CLIENT_ID"),
+      client_secret: System.get_env("FACEBOOK_CLIENT_SECRET")
+    ```
+
+1.  Include the Überauth plug in your controller:
+
+    ```elixir
+    defmodule MyApp.AuthController do
+      use MyApp.Web, :controller
+      plug Ueberauth
+      ...
     end
     ```
+
+1.  Create the request and callback routes if you haven't already:
+
+    ```elixir
+    get "/:provider", AuthController, :request
```

Comment:
Maybe this should show the full `scope`? And mention that this is for use in a Phoenix app?

```elixir
scope "/admin", MyApp do
  pipe_through :browser

  get "/:provider", AuthController, :request
  get "/:provider/callback", AuthController, :callback
end
```